routemodel/data_retrieval/get_weather.py

```diff
@@ -0,0 +1,21 @@
+import sys
+import os.path
+sys.path.append(os.path.dirname(__file__))
+
+from config import WEATHER_API_KEY
+
+one_call = 'https://api.openweathermap.org/data/2.5/onecall?lat=35.6870&lon=-105.9378&exclude=hour,daily&units=metric'
+headers = {'Content-Type': }
+path = '/Users/anhmai/Desktop/weather\ api '
```

Comment:
Can we use relative file paths so that this works everywhere?

For eg. `path = os.path.join(os.path.dirname(__file__),'..','file')`